style.css

```diff
@@ -227,3 +227,17 @@ body .is-layout-flow .wp-block-group {
 	padding-right: 4.5rem;
 	padding-left: 4.5rem;
 }
+
+/*
+ * Pagination styles
+ */
+.wp-block-query-pagination .wp-block-query-pagination-numbers > *:not(:last-child) {
+	margin-right: clamp(1.44rem, 1.91vw + 0.97rem, 2.5rem);
+}
+
+.wp-block-query-pagination .wp-block-query-pagination-numbers > *:not(:last-child) {
+	margin-right: clamp(1.44rem, 1.91vw + 0.97rem, 2.5rem);
+}
```

Comment:
A slip of keyboard probably, this style is added twice

Reply:
Fixed here [0243544](https://github.com/Automattic/sensei-theme/pull/39/commits/0243544893ca778bf5bdd8fde042541721a8da7d)